src/domain/tokens/entities/token-info.entity.ts

```diff
@@ -0,0 +1,15 @@
+export enum TokenType {
+  Erc721 = 'ERC721',
+  Erc20 = 'ERC20',
+  NativeToken = 'NATIVE_TOKEN',
+  Unknown = 'UNKNOWN',
```

Comment:
The Safe Transaction Service does not have an `UNKNOWN` type for `TokenType`.

In the Rust client it's under `#[serde(other)]` which is the fallback in case matching with the other types fail.

Reply:
Fixed in [16eda59](https://github.com/5afe/safe-client-gateway-nest/pull/178/commits/16eda598ca12e2c19b144794bd4a8812dae7d5db)